cookbooks/ambari/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We have the ASF license for the entire repo at the root; I do not believe there is a need for duplication below.